app/js/components/manageApps/ManageApps.jsx

```diff
@@ -119,8 +119,12 @@ export default class ManageApps extends React.Component {
       this.handleApplist();
     }).catch(
       (error) => {
-        location.href = `${location.href.substr(0, location.href.indexOf(location.href.split('/')[4]))}login.htm`;
-        toastr.error(error);
+        error.response.status === 401 ?
+          location.href = `${location.href.substr(0, location.href.indexOf(location.href.split('/')[4]))}login.htm`
+          : error.response.status === 500 ?
+            toastr.error("Some modules failed to start")
```

Comment:
Does 500 mean modules failure to start? Can't it be something else?

Reply:
It would return `Server returned code 500` as the response, which does not inform the user of anything. And when code 500 is returned, some or all the modules are not started, so the response message is more informative.